assets/js/llms-admin.js

```diff
@@ -68,6 +68,7 @@
 				multiple: false,
```

Comment:
add changelog tags to function docblock and file header

Reply:
fixed in https://github.com/gocodebox/lifterlms/pull/832/commits/bc7ca7152db7783d7ca1b91c4471ed8389acb90a